doc/classes/ParallaxLayer.xml

```diff
@@ -13,6 +13,7 @@
 	<members>
 		<member name="motion_mirroring" type="Vector2" setter="set_mirroring" getter="get_mirroring" default="Vector2(0, 0)">
 			The ParallaxLayer's [Texture2D] mirroring. Useful for creating an infinite scrolling background. If an axis is set to [code]0[/code], the [Texture2D] will not be mirrored.
+			If the length of the viewport axis is bigger than 2x the mirrored axis size it will not repeat infinitely, as the parallax layer only has 2 instances of the sprite at any one time.
```

Comment:
```suggestion
			If the length of the viewport axis is bigger than twice the mirrored axis size, it will not repeat infinitely, as the parallax layer only draws 2 instances of the texture at any one time.
```
(Please use `git commit --amend` and `git push --force` when doing changes to keep it all in a single commit ready to merge, see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request)).